mas.py

```
@@ -0,0 +1,104 @@
+def bad_docstring_function(a, b):
+    """
+    This function does something with two numbers.
+    """
+    return a * b
+
+print(bad_docstring_function(2, 3))
+
+# Misuse of `try` block
+def bad_error_handling():
+    try:
+        1 / 0  # Will always raise an exception
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)